modules/nw-metallb-bgp.adoc

```diff
@@ -0,0 +1,63 @@
+When a router receives traffic for the load balancer IP address, the router picks one of the nodes with a `speaker` pod that advertised the IP address.
+The router sends the traffic to that node.
+After traffic enters the node, the service proxy for the CNI network provider distributes the traffic to all the pods for the service.
+
+The directly-connected router on the same layer 2 network segment as the cluster nodes can be configured as a BGP peer.
+If the directly-connected router is not configured as a BGP peer, you need to configure your network so that packets for load balancer IP addresses are routed between the BGP peers and the cluster nodes that run the `speaker` pods.
+
+Each time a router receives new traffic for the load balancer IP address, it creates a new connection to a node.
+Each router manufacturer has an implementation-specific algorithm for choosing which node to initiate the connection with.
+However, the algorithms commonly are designed to distribute traffic across the available nodes for the purpose of balancing the network load.
+
+If a node becomes unavailable, the router initiates a new connection with another node that has a `speaker` pod that advertises the load balancer IP address.
+
+.MetalLB topology diagram for BGP mode
+image::209_OpenShift_BGP_0122.png["Speaker pods on host network 10.0.1.0/24 use BGP to advertise the load balancer IP address, 203.0.113.200, to a router."]
+
+The preceding graphic shows the following concepts related to MetalLB:
+
+* An application is available through a service that has an IPv4 cluster IP on the `172.130.0.0/16` subnet.
+That IP address is accessible from inside the cluster.
+The service also has an external IP address that MetalLB assigned to the service, `203.0.113.200`.
+
+* Nodes 2 and 3 have a pod for the application.
+
+* The `speaker` daemon set runs a pod on each node.
+The MetalLB Operator starts these pods.
+You can configure MetalLB to specify which nodes run the `speaker` pods.
+
+* Each `speaker` pod is a host-networked pod.
+The IP address for the pod is identical to the IP address for the node on the host network.
+
+* Each `speaker` pod starts a BGP session with all BGP peers and advertises the load balancer IP addresses or aggregated routes to the BGP peers.
+The `speaker` pods advertise that they are part of Autonomous System 65010.
+The diagram shows a router, R1, as a BGP peer within the same Autonomous System.
+However, you can configure MetalLB to start BGP sessions with peers that belong to other Autonomous Systems.
+
+* All the nodes with a `speaker` pod that advertises the load balancer IP address can receive traffic for the service.
+
+** If the external traffic policy for the service is set to `cluster`, then all the `speaker` pods advertise the `203.0.113.200` load balancer IP address and all the nodes with a `speaker` pod can receive traffic for the service.
+At least one endpoint for the service must be in the `Ready` condition.
+
```

Comment:
The host prefix is advertised to the peered router only if the external traffic policy is cluster.

Reply:
@asood-rh sounds good, can I say router peer instead of peered router?